Sources/TuistCloud/Services/CloudURLService.swift

```diff
@@ -29,7 +29,7 @@ public final class CloudURLService: CloudURLServicing {
     public init() {}
 
     public func url(serverURL: String?) throws -> URL {
-        let cloudURL = serverURL ?? ProcessInfo.processInfo.environment["TUIST_CLOUD_URL"] ?? Constants.tuistCloudURL
+        let cloudURL = ProcessInfo.processInfo.environment["TUIST_CLOUD_URL"] ?? serverURL ?? Constants.tuistCloudURL
```

Comment:
I'm not sure if the env variable should take precedence over what you have in the config (or whatever we pass in), that's imho surprising. Maybe we should consider having a new env variable that'd be internal only since we are introducing this to override a URL in the CI to upload to canary.